Header header
sensor_msgs/PointCloud2 point_cloud
sensor_msgs/Image color_image
sensor_msgs/Image depth_image
sensor_msgs/CameraInfo camera_info
---
int32 grasp_num 
string server_name
geometry_msgs/Vector3[] approach 
geometry_msgs/PoseStamped[] grasp_pose

